src/components/HomePage.vue

```diff
@@ -50,7 +50,8 @@ export default {
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
 .logo {
-  border-radius: 50%;
+  width: 250px;
```

Comment:
Is it a common practice in web development to set images with fixed width and fixed height? How does that scale in the mobile browsers where there might not be enough space?

Reply:
I agree with you @kgkoutio branch name should contain a descriptive message. 
According to the width and height in web development it is usually used to set a width and a height and then change it in the responsive part. I haven't add the responsive part yet. In the responsive part we can set the size of the image for a Tablet version ,mobile version etc. This is based my knowledge in React framework, I will check if there are differences in Vuejs.
Example: 
/***************   MOBILE VIEW   ***************/
.nav_logo {
    margin: 0 25px 0 10%;
    height: 60px;
  }